### PR TITLE
small change added

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/routes.ts [hunk 1]
```diff
@@ -340,10 +340,13 @@ export class Routes {
      */
     app.get(PAYMENT_GATEWAY_ENTRY_URL, errorHandler(PaymentHandler));
     app.get(PAYMENT_RETURN_URL_CALLBACK, errorHandler(PaymentValidationHandler));
-
-    app.get('/api/v1/session', (req, res) => res.json(req.session));
+    app.get('/api/v1/session', (req, res) => {
```
Comment (vivek-sekhar) [inline src/main/routes.ts:343]: This implementation is not the right approach. You need to make sure the page is responding fast and not load and timeout.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/routes.ts [hunk 2]
```diff
@@ -340,10 +340,12 @@ export class Routes {
      */
     app.get(PAYMENT_GATEWAY_ENTRY_URL, errorHandler(PaymentHandler));
     app.get(PAYMENT_RETURN_URL_CALLBACK, errorHandler(PaymentValidationHandler));
-
-    app.get('/api/v1/session', (req, res) => res.json(req.session));
+    if (app.locals.ENV !== 'production') {
```
Comment (vivek-sekhar) [inline src/main/routes.ts:343]: use enum

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/routes.ts (hunk 1 above)
Comment (SasankAchyuta9) [reply src/main/routes.ts]: Changed

File: src/main/routes.ts (hunk 2 above)
Comment (SasankAchyuta9) [reply src/main/routes.ts]: Done vivek.

File: src/main/routes.ts [hunk 3]
```diff
@@ -340,10 +340,12 @@ export class Routes {
      */
     app.get(PAYMENT_GATEWAY_ENTRY_URL, errorHandler(PaymentHandler));
     app.get(PAYMENT_RETURN_URL_CALLBACK, errorHandler(PaymentValidationHandler));
-
-    app.get('/api/v1/session', (req, res) => res.json(req.session));
+    if (app.locals.ENV !== EnvironmentValue.PRODUCTION) {
```
Comment (vivek-sekhar) [inline src/main/routes.ts:343]: Change EnvironmentValue to Environment

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

**Comment by SasankAchyuta9** [comment]
closing this as we have another one against master

**Pull request closed without merging**